workout.py

```diff
@@ -1,73 +1,89 @@
 #!/usr/bin/env python
-
 import argparse
 import json
-from models import Workout, Exercise, Session, ExerciseCategory, WorkoutExercise
-from random import random, shuffle, sample
+from models import Workout, Exercise, ExerciseCategory, WorkoutExercise
+import yaml
+from random import random, sample
 import subprocess
-from six import iteritems
 from itertools import groupby, count
+from sqlalchemy.orm import sessionmaker, scoped_session
+from sqlalchemy import create_engine
+import logging
+logging.basicConfig(level=logging.INFO)
+
+class GenerateWorkout(object):
```

Comment:
minor nitpick: since it's a class, maybe rename to "WorkoutGenerator" and `generate_workout` to `generate`?